### PR TITLE
Rgulia ossrh

### PR DESCRIPTION
**Pull request by riccardogulia**
This change enables PGP signing through a profile, only when deploying jars.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,33 +82,48 @@
...
+<!--
+    <repositories>
+        <repository>
+            <id>aaa-releases</id>
+            <name>${repositoryName}</name>
+            <url>${releaseServer}/${repositoryName}</url>
+        </repository>
+    </repositories>
+ -->
+
```
Comment (owenkellett) [inline pom.xml]: Remove the commented out sections of the pom.

File: pom.xml [hunk 2]
```diff
@@ -55,33 +82,48 @@
...
+    <!-- The following is to be removed for OSS deployment -->
+<!--
     <pluginRepositories>
         <pluginRepository>
             <id>sonatype-releases</id>
             <name>Sonatype Releases</name>
             <url>https://oss.sonatype.org/content/repositories/releases/</url>
         </pluginRepository>
     </pluginRepositories>
+ -->
```
Comment (owenkellett) [inline pom.xml]: Remove the commented out sections of the pom.

File: examples/here-oauth-client-example/pom.xml [hunk 3]
```diff
@@ -36,6 +37,15 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
+                <version>${maven-gpg-plugin.version}</version>
+            </plugin>
```
Comment (owenkellett) [inline examples/here-oauth-client-example/pom.xml]: Does the gpg plugin do anything if the gpg-sign-artifacts profile is not active?  If not, I don't think you need this here.

File: here-oauth-client/pom.xml [hunk 4]
```diff
@@ -55,6 +56,15 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
+                <version>${maven-gpg-plugin.version}</version>
+            </plugin>
```
Comment (owenkellett) [inline here-oauth-client/pom.xml]: Again, I don't think you need the gpg plugin to be included here.

File: examples/here-oauth-client-example/pom.xml [hunk 5]
```diff
@@ -15,8 +15,9 @@
 
     <licenses>
         <license>
-            <name>Apache License 2.0</name>
-            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+          <name>Apache License, Version 2.0</name>
+          <url>https://github.com/heremaps/here-aaa-java-sdk/blob/master/LICENSE</url>
+          <distribution>repo</distribution>
```
Comment (owenkellett) [inline examples/here-oauth-client-example/pom.xml]: The licenses section does not need to be duplicated in the child poms if it is in the parent pom.

File: here-oauth-client-dist/pom.xml [hunk 6]
```diff
@@ -13,12 +13,13 @@
...
-            <url>http://www.here.com</url>
-        </license>
-    </licenses> 
+	<licenses>
+		<license>
+		  <name>Apache License, Version 2.0</name>
+		  <url>https://github.com/heremaps/here-aaa-java-sdk/blob/master/LICENSE</url>
+		  <distribution>repo</distribution>
+		</license>
+	</licenses>
```
Comment (owenkellett) [inline here-oauth-client-dist/pom.xml]: The licenses section does not need to be duplicated in the child poms if it is in the parent pom.

File: here-oauth-client/pom.xml [hunk 7]
```diff
@@ -13,12 +13,13 @@
...
-            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-        </license>
-    </licenses>
+	<licenses>
+		<license>
+		  <name>Apache License, Version 2.0</name>
+		  <url>https://github.com/heremaps/here-aaa-java-sdk/blob/master/LICENSE</url>
+		  <distribution>repo</distribution>
+		</license>
+	</licenses>
```
Comment (owenkellett) [inline here-oauth-client/pom.xml]: The licenses section does not need to be duplicated in the child poms if it is in the parent pom.

**Comment by rgulia** [comment]
Thank you for your review. I have included all of the changes that you requested and submitted for build

**Pull request merged**